dune-project

```diff
@@ -368,6 +368,10 @@ functions.
     (process_limits (>= "1.5.5"))
   )
 )
+(package (name source)
+  (synopsis "XXX")
```

Comment:
Might as well actually write something here no?

Reply:
Done in af778e761cbbc7218c2a55ec906a7ed1b3aebd93

Reply:
I still see XXX.

Reply:
I was just waiting to finish addressing Matthew's comments to push the changes. It should be present now.